templates/taxi/driver_list.html

```diff
@@ -7,7 +7,7 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Consider using single quotes in the `url` tag to maintain consistency with Django's template syntax: `{% url 'taxi:driver-detail' pk=driver.id %}`.